examples/data-sources/kentik-synthetics_agent/provider.tf

```diff
@@ -1,8 +1,9 @@
+# Note: configuration for locally-built provider
```

Comment:
Can the file be shared (somehow) among the examples, so that we do not need to have several identical copies?

Reply:
Symlink are the way to do that. Updated description  https://clkentik.atlassian.net/browse/KNTK-243